src/me/ryanhamshire/PopulationDensity/PopulationDensity.java

```diff
@@ -535,7 +536,7 @@ public void onEnable()
         try
         {
             Metrics metrics = new Metrics(this, 3343);
-            metrics.addCustomChart(new Metrics.SimplePie("bukkit_impl", this::getVersionImplementation));
+            metrics.addCustomChart(new SimplePie("bukkit_impl", this::getVersionImplementation));
```

Comment:
I guess this is ok, kind of a style thing I guess.

Reply:
no, in the newest version SimplePie ist not a static Class of Metrics anymore. It is now part of org.bstats.charts

Reply:
<img width="595" alt="image" src="https://github.com/user-attachments/assets/a5834e60-a9ea-491e-bd9e-86f1bbf062ec">

old version

<img width="1010" alt="image" src="https://github.com/user-attachments/assets/fb7f56b9-964a-47ba-a201-1fc1a4f79d65">

Reply:
Ah, good to know thank you!